### ISSUE TITLE
Node 10 v8 compilation error (toNumber() deprecation)

### ISSUE THREAD

**Issue by ekarak** [OpenZWave/node-openzwave-shared#233]
```../src/openzwave-config.cc:66:35: error: no matching function for call to ‘v8::Value::ToNumber()’
   uint8 param = info[1]->ToNumber()->Value();
...
../src/openzwave-config.cc: In static member function ‘static Nan::NAN_METHOD_RETURN_TYPE OZW::OZW::RequestAllConfigParams(Nan::NAN_METHOD_ARGS_TYPE)’:
../src/openzwave-config.cc:81:36: error: no matching function for call to ‘v8::Value::ToNumber()’
   uint8 nodeid = info[0]->ToNumber()->Value();
                                    ^
In file included from ../src/openzwave.hpp:26:0,
                 from ../src/openzwave-config.cc:18:
/home/pi/.node-gyp/10.1.0/include/node/v8.h:2366:44: note: candidate: v8::MaybeLocal<v8::Number> v8::Value::ToNumber(v8::Local<v8::Context>) const
   V8_WARN_UNUSED_RESULT MaybeLocal<Number> ToNumber(
                                            ^~~~~~~~
```

**Comment by ekarak** [comment]
also we need to take care of this:
```../src/openzwave.cc: In function ‘void OZW::init(v8::Handle<v8::Object>, v8::Handle<v8::Object>)’:
../src/openzwave.cc:43:3: warning: ‘v8::String::Utf8Value::Utf8Value(v8::Local<v8::Value>)’ is deprecated: Use Isolate version [-Wdeprecated-declarations]
   ));
   ^
In file included from /home/pi/.node-gyp/10.1.0/include/node/v8.h:26:0,
                 from ../src/openzwave.hpp:26,
                 from ../src/openzwave.cc:18:
/home/pi/.node-gyp/10.1.0/include/node/v8.h:2822:28: note: declared here
                   explicit Utf8Value(Local<v8::Value> obj));
```

**Comment by ekarak** [comment]
and this ```../src/callbacks.cc:174:49: warning: ‘v8::Local<v8::Value> Nan::Callback::Call(v8::Local<v8::Object>, int, v8::Local<v8::Value>*) const’ is deprecated (declared at /home/travis/build/OpenZWave/node-openzwave-shared/node_modules/nan/nan.h:1596) [-Wdeprecated-declarations]
     emit_cb->Call(Nan::New(ctx_obj), 4, emitinfo);```

**Comment by ekarak** [comment]
released version 1.4.5, resolving
